.github/workflows/run-e2e-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Run E2E tests
+on:
+  schedule:
+    # runs tests every day at 4am
+    - cron: '0 4 * * *'
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1 2 3]
```

Comment:
Runs the test in parallel in 3 jobs and submits the results to the cypress dashboard